kratos/tests/cpp_tests/spaces/test_kratos_space.cpp

```diff
@@ -0,0 +1,77 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Riccardo Rossi
+//                   Philipp Bucher (https://github.com/philbucher)
+//
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "includes/define.h"
+#include "testing/testing.h"
+#include "spaces/kratos_space.h"
+
+
+namespace Kratos {
+namespace Testing {
+
+
+KRATOS_TEST_CASE_IN_SUITE(KratosSpaceNormSparseMatrix, KratosCoreFastSuite)
+{
+    typedef std::size_t IndexType;
+    typedef KratosSpace<double, CsrMatrix<double,IndexType> , SystemVector<double,IndexType>> SparseSpaceType;
```

Comment:
Why not creating a specific `namespace KratosSpaceTesting` inside the `Testing` one to avoid declaring this in each test?

Reply:
Or an unnamed namespace